docs/source/advanced.rst

```diff
@@ -130,19 +139,23 @@ You can see command line documentation by running ``onionshare --help``::
     --disable_csp             Publish website: Disable Content Security Policy header (allows your website to use third-party resources)
     -v, --verbose             Log OnionShare errors to stdout, and web errors to disk
 
-Legacy addresses
+Legacy Addresses
 ----------------
 
-OnionShare uses v3 Tor onion services by default. These are modern onion addresses that have 56 characters, for example::
+OnionShare uses v3 Tor onion services by default.
```

Comment:
I think this is called something like v3 hidden Tor long onion something something, and I ran out of steam to check up on it.